e2e/testdata/fn-render/basicpipeline-image-short-path/.expected/diff.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/resources.yaml b/resources.yaml
```

Comment:
same comment as previous (with same name corresponding to fn-eval test): `short-image-path`